### DISCUSSION TITLE
Why Budgie over Plasma or other platforms?

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#382] [General]
Haven't taken a drive with Budgie for a few years now.
Think last time was a few years back with Solus.

Since then, KDE Plasma has been my fav.
Easy to customize (love global themes), software repo. is excellent, find files in a flash via super-key and is super low on resources with SpiralLinux version.

**Comment by AlphaElwedritsch** [comment]
Well, what can you say? Choosing the DE is always a matter of personal taste.
There is no better or worse.
I now use Gnome and my mother's laptop runs Budgie.
For example, I was never really able to enjoy KDE.
everyone can take what they want. The good thing is that you have a choice and you don't have to take and accept what is presented to you like with Windows.

**Comment by dacomboman** [comment]
I understand.

But if you can share a bit about what makes Budgie your fav., that would be nice to hear.

 

-Daniel

**Comment by AlphaElwedritsch** [comment]
What can I say, just the look and feel.
The simplicity. the reduction to the minimum.
gnome is even easier than budgie.
I hate a system that takes me days to configure.
I don't do much either. always stay as close to stock as possible. preferably vanilla

**Comment by dacomboman** [comment]
Wish i could be like that!

Always start by saying, "okay, lets leave the default setup for a while".
Then, sooner than later, start itching to change things around...
It's like going from Firefox (boring, a tinge slow, but practical) to Vivaldi (sparkles, fast, distractions).
Not the best comparison, but you get the drift.

**Comment by AlphaElwedritsch** [comment]
Hahaha, yes Vivaldi
A monster of browser...
Thousand of switches and parameters...
Colors here, colors there, up, down, left, right, square, round and what else I know.
but no options regarding security, privacy, anti fingerprinting, tracking etc...

**Reply by dacomboman** [reply]
True.

But (oddly enough) via chrome://settings (which somehow switch to read vivaldi://settings after) can set DNS to Cloudflare and others.

**Comment by AlphaElwedritsch** [comment]
And the cool thing is that @geckolinux  in SpiralLinux configures the system as such and the DEs exactly as you need them. not too much, not too little. precisely.
You no longer have to do anything yourself.

**Comment by dacomboman** [comment]
In a way, Vivaldi reflects the KDE Plasma ambience.
"Simple by Default, Powerful when Needed."
Glancing over all possible options at once, might take away that simplicity.

It just becomes a tug of war between wanting to keep things simple and the desire to tinker with, enhance or tweak things...

Following Tom (of SwitchedToLinux fame), i keep at least 2 or 3 browsers on hand.

**Comment by AlphaElwedritsch** [comment]
I just tried KDE again in a Boxes VM. But as I suspected. You will be overwhelmed by the setting options. That's just not my thing.
I'd rather stick with Gnome/Budgie. A minimal jungle of setting options.

**Comment by dacomboman** [comment]
Wait until you see changes in Plasma 6!

Guess it's my turn to take Budgie out for a spin.

**Comment by dacomboman** [comment]
@AlphaElwedritsch - replaced KDE with Budgie - seems to use more resources than KDE, catfish never finds anything for me and i can't figure out how to get FSearch on bottom panel and software installer only seems to have Flatpaks.

Now i remember why i prefer KDE and even Cinnamon.

**Reply by AlphaElwedritsch** [reply]
The tastes are so different ;)

**Reply by dacomboman** [reply]
And that's okay! ;)

It's what Linux is all about, right?
